package/batocera/core/batocera-configgen/configgen/configgen/Command.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, array, env=dict()):
     def __str__(self):
         str = list()
 
-        for varName, varValue in self.env.items():
+        for varName, varValue in list(self.env.items()):
```

Comment:
Did you use 2to3? I'm surprised about this `list`

Reply:
yes have use for try to see is ok or not